File: SDKSampleJava/app/src/main/java/com/gyant/sdksamplejava/MainActivity.java
package com.gyant.sdksamplejava;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void changeToFragmentPage(View view) {
        Intent intent = new Intent(this, DisplayFragmentActivity.class);
        startActivity(intent);
    }
    public void changeToViewPage(View view) {
        Intent intent = new Intent(this, DisplayGyantViewActivity.class);
        startActivity(intent);
    }

    public void changeToActivityPage(View view) {
        Intent intent = new Intent(this, DisplayGyantChatActivity.class);
        startActivity(intent);
    }

    public void changeToViewWithThemePage(View view) {
        Intent intent = new Intent(this, DisplayGyantViewWithThemeActivity.class);
        startActivity(intent);
    }

    public void changeToViewWithMessageListener(View view) {
        Intent intent = new Intent(this, DisplayGyantViewWithMessageListenerActivity.class);
        startActivity(intent);
    }

    public void changeToViewWithPushToken(View view) {
        Intent intent = new Intent(this, DisplayGyantViewWithPushTokenActivity.class);
        startActivity(intent);
    }
}

File: SDKSampleKotlin/app/src/main/java/com/gyant/sdksamplekotlin/MainActivity.kt
package com.gyant.sdksamplekotlin

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


    }

    fun changeToFragmentPage(view: View) {
        val intent = Intent(this, DisplayFragmentActivity::class.java)
        startActivity(intent)
    }

    fun changeToViewPage(view: View) {
        val intent = Intent(this, DisplayGyantViewActivity::class.java)
        startActivity(intent)
    }

    fun changeToActivityPage(view: View) {
        val intent = Intent(this, DisplayGyantChatActivity::class.java)
        startActivity(intent)
    }

}
